src/agenda/reservation-list/index.js

```diff
@@ -30,13 +30,20 @@ class ReactComp extends Component {
     // considered that the date in question is not yet loaded
     reservations: PropTypes.object,
 
+    // specify a flag to show the left day on the left of reservation
+    showDate: PropTypes.bool,
+
     selectedDay: PropTypes.instanceOf(XDate),
     topDay: PropTypes.instanceOf(XDate),
     refreshControl: PropTypes.element,
     refreshing: PropTypes.bool,
     onRefresh: PropTypes.func,
   };
 
+  static defaultProps = {
+    showDate: true
```

Comment:
If you called the prop 'hideDate' you shouldn't have to set the default to true, plus it makes more sense because you want the option to hide the date...